frontend/src/v5/ui/routes/dashboard/viewerCanvases/viewerCanvases.component.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ViewerCanvas as Canvas3D } from '@/v4/routes/viewerCanvas';
```

Comment:
The 3d toolbar gets updated in a weird way and inconsistent with the 2d side

<img width="596" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/8a408957-e69e-4247-9666-bc14939fc732">

Reply:
- The handle for resizing should only appear on hover 
- The handle for resizing should appear at the y position of the mouse following it.
- The cursor shouldnt change because it s the resizing handle the component which gives resizing feedback,

Reply:
This crashing already happens with the 3d viewer if you resize the page too fast. Will let @carmenfan test and see what should be done

Reply:
Mentioned this before to @sanmont3drepo as a possible solution, could we make it so it only resizes when the user releases the button? A bit like what visual studio is doing here:
![24-05-22 06-19-22](https://github.com/3drepo/3drepo.io/assets/11945337/ebb1272f-417d-4081-8359-6927cb982366)